app/src/main/java/org/cru/godtools/util/ViewUtils.kt

```diff
@@ -16,3 +26,26 @@ fun PicassoImageView?.bindLocalImage(attachment: Attachment?) =
     bindLocalImage(attachment?.takeIf { it.isDownloaded }?.localFileName)
 
 fun PicassoImageView?.bindLocalImage(filename: String?) = this?.setPicassoFile(context.getGodToolsFile(filename))
+
+fun TextView.setTextViewWebLinksForAnalytics(
```

Comment:
I like this better, I want to dig into this logic a bit before I have a more in depth review

Reply:
@frett OK I am cleaning up the conflict right now.